docs/build/messages/messages.md

```diff
@@ -345,70 +345,105 @@ for await (const page of conversation.messages(limit: 25)) {
 </TabItem>
 </Tabs>
 
-## Handle an unsupported content type error
+## Handle unsupported content types
 
-As more [custom](/docs/concepts/content-types#create-a-custom-content-type) and [standards-track](/docs/concepts/content-types#standards-track-content-types) content types enter the XMTP ecosystem, your app might receive a content type your app doesn't support. This could crash your app.
+As more [custom](/docs/concepts/content-types#create-a-custom-content-type) and [standards-track](/docs/concepts/content-types#standards-track-content-types) content types are introduced into the XMTP ecosystem, your app may encounter content types it does not support. This situation, if not handled properly, could lead to app crashes.
+
+Each content type is accompanied by a `contentFallback` property, which offers a descriptive string representing the content type's expected value. It's important to note that content fallbacks are immutable and are predefined in network messages. In instances where `contentFallback` is `undefined`, it indicates that the content, such as read receipts, is not intended to be rendered. If you're venturing into creating custom content types, you're provided with the flexibility to specify a custom fallback. For a deeper dive into this, consider exploring the [Custom Content Type Tutorial](/docs/tutorials/custom-ct).
```

Comment:
> Each content type is accompanied by a `contentFallback` property

should this say each message? or is this referring to the codecs, which have a `fallback` property?